### ISSUE TITLE
Help/bug regarding sort functionality

### ISSUE THREAD

**Issue by coder-amogh** [giotiskl/Filterizr#143]
[I don't know if there's already an existing solution to this or this is a bug. Please move this post from issue to bug if you feel so.]

Filterizr version: I am using Filterizr version 2.2.4 from a CDN. 

Link: https://cdnjs.cloudflare.com/ajax/libs/filterizr/2.2.4/jquery.filterizr.min.js

I have a webpage where in the Bootstrap cards are created dynamically using jQuery after an AJAX request is successful. Basically, server returns a response and jQuery creates HTML code and appends it to the card, when it's done, it instantiates the Filterizr so as to give the user options to sort and the created cards if needed.

The sort functionality is a bit problematic. I have an attribute in the .card.filtr-item called as data-view-count. The problem is when I sort it ascending it seems to sort them assuming they are strings and not integers (or issue related to something like that).

Is there any way to explicitly tell Filterizr to assume them integers and then sort them? 

It would make sense if you look at this example:

Suppose I have 6 cards with .filtr-item class in each. Each of them have one number inside the data-view-count attribute:

[99, 100, 98, 102, 103, 116]

In other words, the first div would get the data-view-count="99", second one would be data-view-count="100" and so on.

Now, when I sort them as ascending, what I expect to happen is this:

[98, 99, 100, 102, 103, 116]

What it turns is this:

[100, 102, 103, 116, 98, 99]

So I think what may be happening is the attribute is considered to be a string and then it's sorted that way.

Does anyone have any idea how I can solve this problem?

I can use data-sort, but there are a lot of attributes (view-count, download-count, rating-total, etc,) which means I can't use data-sort here. I clearly need a way to tell Filterizr to consider them as integers and NOT strings.

Please help! I just love your library and your work :)

**Comment by giannisftaras** [comment]
I can confirm the same issue on a self hosted - local instance, version 2.2.4.

**Comment by coder-amogh** [comment]
> I can confirm the same issue on a self hosted - local instance, version 2.2.4.

Ah, no worries. I left padded the values with 0 and it's all good. I'm closing the issue! :)
